### DISCUSSION TITLE
Source code based is which pre-complied version?

### DISCUSSION THREAD

**Discussion by Onejk1** [Jyers/Marlin#145] [Q&A] [answered]
Just wondering which pre-compiled version the source code is for? having a look at making a few mods and just wondered what I shoudl take the source code as being based on.

**Comment by amandamana** [comment, marked as answer]
All of the pre-compiled versions are based on the same _Ender_3_V2_Extensible_UI_ branch source code. You can replicate the builds yourself by choosing a folder in _Configuration Files/E3V2 Templates/_ and copying the files inside (_Configuration.h_ and _Configuration_adv.h_) to overwrite those same files in _Marlin/_

**Reply by Onejk1** [reply]
Brilliant, thank you. That is exactly what i wanted to know
